features/support/hooks/hooks.blocks.js

```diff
@@ -0,0 +1,38 @@
+import {
```

Comment:
the file format is different to the other files, including the other hooks file, you could mantain the same format. Maybe they could check that and say something to us respect of this.

Reply:
Fixed